### DISCUSSION TITLE
Label text may be unintentionally omitted when <requestedDisplayResolution> is high.

### DISCUSSION THREAD

**Discussion by uregishikoa** [airsdk/Adobe-Runtime-Support#2802] [Q&A] [answered]
On Mac Book with Retina display, if `<requestedDisplayResolution>high</requestedDisplayResolution>` is specified in the manifest, some labels are omitted from the display.

In the following image, the string specified in the `labels` property of `<mx:HSlider>` is omitted. (First image)
Not omitted if `<requestedDisplayResolution>standard</requestedDisplayResolution>` is specified. (Second image)
It is also not omitted when the number of characters in the label is set to something other than 3.

Is there any way to avoid omitting without changing the `<requestedDisplayResolution>` setting or the number of characters in the label?

Environment:
AIR SDK:50.1.1
Flex SDK:4.16

<img width="343" alt="high" src="https://github.com/airsdk/Adobe-Runtime-Support/assets/142779777/fa63fdb8-26e2-4f27-a2d8-2a5d1efee527">
<img width="331" alt="standard" src="https://github.com/airsdk/Adobe-Runtime-Support/assets/142779777/2be2593e-a182-4b06-aa57-d125b8ab8aac">

**Comment by ajwfrost** [comment, marked as answer]
Hi

This looks like it's the same issue as #2645  .. which I can't see in the release notes, but we did have a fix go in to the code in May. Which means it should have been in AIR SDK 50.2.2.6 .. are you able to update to this or a later runtime and try it?

thanks

**Reply by uregishikoa** [reply]
I updated to AIR SDK 50.2.3 and the problem no longer occurs.
Thanks for the quick response.
